Classes/Controller/AbstractController.php

```diff
@@ -43,6 +46,16 @@ public function injectExtConf(ExtConf $extConf): void
         $this->extConf = $extConf;
     }
 
+    public function getQueryBuilderForTable(string $table)
```

Comment:
define return type QueryBuilder

Reply:
I had rebased origin/main. Since than I got these curious methods everywhere in code. DB in controller?!?! Don't understand that. I have removed both methods